config/tests/samples/create/harness.go

```diff
@@ -650,10 +652,7 @@ func MaybeSkip(t *testing.T, name string, resources []*unstructured.Unstructured
 			case schema.GroupKind{Group: "servicenetworking.cnrm.cloud.google.com", Kind: "ServiceNetworkingConnection"}:
 
 			case schema.GroupKind{Group: "serviceusage.cnrm.cloud.google.com", Kind: "Service"}:
-				if os.Getenv("GOLDEN_REQUEST_CHECKS") != "" {
-					// no golden log for this resource  yet
-					t.Skipf("gk %v/%v does not support golden request check; skipping", gvk.GroupKind(), name)
-				}
```

Comment:
Not sure why we need to comment out [this originally](https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/1355). But it skips the `cloudbuildworkerpool` mocktest which has a networkservice dependency.  

I see that we actually have the mockservice. If I remove the skip, will it cause any problem to the LLM resource? @acpana

Reply:
I agree that we should try to avoid exceptions in our tests, it makes them difficult to understand.  Better to use env vars and specify tests explicitly.

Ideally MaybeSkip would not exist, once we have full mockgcp coverage!

The good thing about this is that we call t.Skipf, so it should be _silent_